include/bluetooth/services/bms.h

```diff
@@ -22,29 +22,29 @@ extern "C" {
 
 /** @brief BMS Control Point operation codes (LE transport). */
 enum bt_bms_op {
-	/* Initiates the procedure to delete the bond of
+	/** Initiates the procedure to delete the bond of
 	 * the requesting device.
 	 */
 	BT_BMS_OP_DEL_REQ_BOND = 0x03,
 
-	/* Initiates the procedure to delete all bonds on the device. */
+	/** Initiates the procedure to delete all bonds on the device. */
 	BT_BMS_OP_DEL_ALL_BONDS = 0x06,
 
-	/* Initiates the procedure to delete all bonds except for the one of
+	/** Initiates the procedure to delete all bonds except for the one of
 	 * the requesting device.
```

Comment:
```suggestion
	/** Initiates the procedure to delete all bonds except for the one of
	 *  the requesting device.
```
I think it looks better when the lines are aligned.
Also in other places.

Reply:
Hmm good idea, looks better also in my opinion